jetty-server/src/main/java/org/eclipse/jetty/server/HttpConfiguration.java

```diff
@@ -318,11 +318,21 @@ public boolean getSendXPoweredBy()
         return _sendXPoweredBy;
     }
 
+    /**
+     * Indicates if the {@code Date} header should be sent in responses.
+     *
+     * @param sendDateHeader true if the {@code Date} header should be sent in responses
+     */
     public void setSendDateHeader(boolean sendDateHeader)
     {
         _sendDateHeader = sendDateHeader;
     }
 
+    /**
+     * Indicates if the {@code Date} header will be sent in responses.
+     *
```

Comment:
Some language that indicates that this has a default value of `true` is needed here.

Reply:
Updated. Please review.

Reply:
Meh, it works for me. :-)